app/docs/config.json

```
@@ -0,0 +1,122 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cosmos SDK - Legacy REST and gRPC Gateway docs",
```

We can also update the swagger title to Crypto.org Chain?
```suggestion
    "title": "Crypto.org Chain - Legacy REST and gRPC Gateway docs",
```

---

and this will be reflected on the generated swagger UI right?

---

yeah, seems to be the case